proposed/links.md

```diff
@@ -33,6 +33,43 @@ A Hypermedia Link consists of, at minimum:
 Various other attributes of the Link may exist, depending on the format used. As additional attributes 
 are not well-standardized or universal, this specification does not seek to standardize them.
 
+For the purposes of this specification, the following definitions apply.
+
+*    **Implementing Object** -An object that implements one of the interfaces
```

Comment:
It's all done manually and I don't think we have a standard for where we wrap, so... meh. I hate manually wrapping text.  I hate Markdown.